README.md

```diff
@@ -30,6 +30,15 @@ ansible-playbook -i environments/$ENV/inventory main.yml
 
 Please see the environment's README for specific details.
 
+## Updating generated content
+
+This project has generatated content in it. By running the `python3 .update/do.py` you are updating that content. It's your responsibilty to commit, push and create a pull request for the changes.
+
+```sh
+pip3 install --upgrade -r .update/requirements.txt
+python3 .update/do.py update-rex-redirects
```

Comment:
Doesn't it also need the `rex-host` arg?

Reply:
Sure does, but we don't know what that is yet. Note, this PR is in draft state.

Reply:
I'll put a placeholder in for now...